modules/renter/contractor/uptime_test.go

```diff
@@ -104,12 +104,20 @@ func TestIntegrationReplaceOffline(t *testing.T) {
 // mapHostDB is a hostDB that implements the Host method via a simple map.
 type mapHostDB struct {
 	stubHostDB
-	hosts map[modules.NetAddress]modules.HostDBEntry
+	hosts map[types.FileContractID]modules.HostDBEntry
 }
 
 func (m mapHostDB) Host(addr modules.NetAddress) (modules.HostDBEntry, bool) {
-	h, exists := m.hosts[addr]
-	return h, exists
+	if len(m.hosts) == 0 {
+		return modules.HostDBEntry{}, false
+	}
+	if len(m.hosts) != 1 {
+		panic("oops")
+	}
+	for _, host := range m.hosts {
+		return host, true
+	}
+	return modules.HostDBEntry{}, false
```

Comment:
hosts in the test don't have an addr anymore. So, no matter what addr you pass to the function, it'll return the only host in the map, panicking if there's more than one host in the map.

it's hacky